observability/prometheus-backend/src/main/scala/sttp/client3/prometheus/PrometheusBackend.scala

```diff
@@ -41,13 +42,17 @@ object PrometheusBackend {
         delegate,
         RequestListener.lift(
           new PrometheusListener(
-            requestToHistogramNameMapper,
-            requestToInProgressGaugeNameMapper,
-            (rr: (Request[_, _], Response[_])) => responseToSuccessCounterMapper(rr._1, rr._2),
-            (rr: (Request[_, _], Response[_])) => responseToErrorCounterMapper(rr._1, rr._2),
-            (r: (Request[_, _], Throwable)) => requestToFailureCounterMapper(r._1, r._2),
-            requestToSizeSummaryMapper,
-            (rr: (Request[_, _], Response[_])) => responseToSizeSummaryMapper(rr._1, rr._2),
+            (req: Request[_, _]) => requestToHistogramNameMapper(req).map(addLabelPairs(_, req, None)),
```

Comment:
as I understand, this will *always* add the labels (if they are not present). So it's not possible not to add them - if somebody, for some reason would want that.

Maybe, instead we could modify the default values of parameters in `apply` so that the user has chance to fully override the behavior?

Reply:
Done - https://github.com/softwaremill/sttp/pull/1651/commits/ec2ca4a498ea0dcc852c704541e45ed824eefaf7